sklearn/ensemble/_forest.py

```diff
@@ -1475,17 +1499,23 @@ class RandomForestRegressor(ForestRegressor):
         the input samples) required to be at a leaf node. Samples have
         equal weight when sample_weight is not provided.
 
-    max_features : {"auto", "sqrt", "log2"}, int or float, default="auto"
+    max_features : {"sqrt", "log2", None}, int or float, default=1.0
         The number of features to consider when looking for the best split:
 
         - If int, then consider `max_features` features at each split.
         - If float, then `max_features` is a fraction and
           `round(max_features * n_features)` features are considered at each
           split.
         - If "auto", then `max_features=n_features`.
-        - If "sqrt", then `max_features=sqrt(n_features)`.
+        - If "sqrt", then `max_features=sqrt(n_features)` (same as "auto").
         - If "log2", then `max_features=log2(n_features)`.
-        - If None, then `max_features=n_features`.
+        - If None or 1.0, then `max_features=n_features`.
+
+        Note: The default of 1.0 is equivalent to bagged trees and more
+        randomness can be achieved by setting smaller values, e.g. 0.3.
+
+        .. deprecated:: 1.0
+            Previous default option "auto" has been deprecated.
```

Comment:
I think that we need two entries here. One to highlight the change of default and another for the deprecation:

```
.. versionchanged:: 1.0
   The default of `max_features` changed from `"auto"` to 1.0.

.. deprecated:: 1.0
   The `"auto"` option was deprecated in 1.0 and will be removed
   in 1.2
```

Could you change for the different entry of all estimators?